src/components/ReceiverInput.tsx

```diff
@@ -138,6 +138,7 @@ function ReceiverInput({ setError }: Props) {
       <Box display="flex" alignItems="center">
         <AccountIdenticon address={addressInput} formatFound={formatFound} />
         <InputBase
+          id="receiver_input"
```

Comment:
Let's use `_` or `-` and `test-` prefix consistently.
```suggestion
          id="test-receiver-input"
```